### PR TITLE
Make internal and external namespace more consistent

### PR DESCRIPTION
**Pull request by ekechinwokah**
See https://github.com/infinohq/infino/issues/128

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -762,22 +762,22 @@ mod tests {
...
-    let mut data_points_received: Vec<DataPoint> = serde_json::from_slice(&body).unwrap();
+    let mut metric_points_received: Vec<MetricPoint> = serde_json::from_slice(&body).unwrap();
 
-    check_data_point_vectors(&data_points_expected, &data_points_received);
+    check_metric_point_vectors(&metric_points_expected, &metric_points_received);
 
     // Sleep for 2 seconds and refresh from the index directory.
     sleep(Duration::from_millis(2000)).await;
 
-    let refreshed_tsldb = Tsldb::refresh(config_dir_path);
```
Comment (vinaykakade) [inline src/main.rs:772]: Let us keep search_ts / search_query around (until new client versions are published, new docker container is published) for some time for backwards compatibility.

File: clients/infinopy/infinopy/client.py [hunk 2]
```diff
@@ -26,8 +26,8 @@ def append_ts(self, payload):
         path = "/append_ts"
         return self._request("POST", path, json=payload)
 
-    def search_log(self, text, start_time=None, end_time=None):
```
Comment (vinaykakade) [inline clients/infinopy/infinopy/client.py:29]: Please update the version in setup.py

**Comment by vinaykakade** [comment]
Thanks for the changes. Minor comments added to make this backwards compatible. Pl check that all the tests and dashboard ones pass (`make test` in infino, infino/clients/infinopy and infino/dashboard).

Will merge this on Sat night IST / Sat morning PST.

File: src/main.rs (hunk 1 above)
Comment (ekechinwokah) [reply src/main.rs:772]: done

File: clients/infinopy/infinopy/client.py (hunk 2 above)
Comment (ekechinwokah) [reply clients/infinopy/infinopy/client.py:29]: updated to 0.0.6

**Comment by ekechinwokah** [comment]
done and passes tests. once approved and merged let me know and i will publish the infinopy client package. i've only tested on mac os so please let me know asap if anything breaks.

**Pull request merged**